ci_scripts/ci_post_clone_sub_scripts/googleservice-info.sh

```
@@ -0,0 +1,43 @@
+#! /bin/bash
+
+# 스크립트 실행 위치를 고정
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+cd "$SCRIPT_DIR/../.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)